challenges/classes.js

```diff
@@ -1,11 +1,20 @@
 // Copy and paste your prototype in here and refactor into class syntax.
+let CuboidMaker = {
+  constructor(properties) {
+    this.properties = length.properties;
```

Comment:
* The proper way to access the length property of the `properties` object would be like this:
```this.length = properties.length```
* On your code you are overwriting the properties inside your class every time, and you are length, width, and height are undefined.